sdcm/utils/aws_region.py

```diff
@@ -227,11 +237,52 @@ def create_sct_security_group(self):
                             }
                         ]
                     },
+                ]
+            )
+
+    def get_sct_test_security_group(self, test_id) -> EC2ServiceResource.SecurityGroup:
+        name = self.SCT_TEST_SECURITY_GROUP_NAME_TMPL.format(test_id)
+        security_groups = self.client.describe_security_groups(Filters=[{"Name": "tag:Name",
+                                                                         "Values": [name]},
+                                                                        {"Name": "tag:TestId",
+                                                                         "Values": [test_id]
+                                                                         }])
+        LOGGER.debug("Found Security Groups: %s", security_groups)
+        existing_sgs = security_groups.get("SecurityGroups", [])
+        if len(existing_sgs) == 0:
+            return None
+        assert len(existing_sgs) == 1, \
+            f"More than 1 Security group with {name} found " \
+            f"in {self.region_name}: {existing_sgs}!"
+        return self.resource.SecurityGroup(existing_sgs[0]["GroupId"])  # pylint: disable=no-member
+
+    def provide_sct_test_security_group(self, test_id: str):
```

Comment:
This `test_security_group` is about opening all the related ports to public?
Line of comment would be helpful.

Cleanup resources cleans this sg?

Reply:
yes, cleanup does clean it up.

Reply:
Also added a docstring to this function, explaining when this should be used.